chirp/drivers/anytone.py

```diff
@@ -381,7 +381,7 @@ class AnyTone5888UVRadio(chirp_common.CloneModeRadio,
     VENDOR = "AnyTone"
     MODEL = "5888UV"
     BAUD_RATE = 9600
-    _file_ident = ["QX588UV", "588UVN"]
+    _file_ident = ["QX588UV".encode(), "588UVN".encode()]
```

Comment:
Why not use `b"QX588UV"` to be consistent with everything else? Static strings should be defined as byte strings to avoid the runtime overhead of encoding them. There are a lot of examples here, so it would be good if we can get that fixed first.

Reply:
Done!